### ISSUE TITLE
Insufficient contrast for some Card text in docs?

### ISSUE THREAD

**Issue by cvrebert** [twbs/bootstrap#17287]
I'm guessing that the "Someone famous in Source Title"  text in http://v4-alpha.getbootstrap.com/components/card/#background-variants lacks sufficient contrast to pass accessibility guidelines?
CC: @patrickhlauke

**Comment by patrickhlauke** [comment]
yes for all of the examples (color contrast hovering between 1.5:1 and 2.4:1 (when it should be at least 4.5:1))

**Comment by kkirsche** [comment]
@patrickhlauke what accessibility guidelines? Section 508 compliance or something else?

**Comment by zacechola** [comment]
@kkirsche [WCAG 2.0](http://www.w3.org/TR/WCAG20/). 508 of the Rehabilitation Act is a US-specific law, which does a fair amount of mapping to the w3 recommendation.

**Comment by bassjobsen** [comment]
maybe https://github.com/twbs/bootstrap/pull/18478 can help? Also see: https://github.com/bassjobsen/bootstrap-a11y-theme

**Comment by mdo** [comment]
The text was updated awhile back i believe, so closing this out as resolved.
